staging/src/k8s.io/api/OWNERS

```diff
@@ -1,51 +1,11 @@
 approvers:
-- erictune
-- lavalamp
-- smarterclayton
-- thockin
-- liggitt
-# - bgrant0607 # manual escalations only
+- api-approvers
```

Comment:
for staging components pushed out to other repos, the OWNERS_ALIASES file in the root will be lost. not sure we want to reference that here

Reply:
Does that matter, since no one should be making changes in those repos? (Should we be copying OWNERS files at all?)

Reply:
The goal is to shape them so they can be standalone repos, even if dev and CI haven't caught up to allow that yet

Reply:
Reverted.